bndtools.api/src/org/bndtools/api/editor/package-info.java

```diff
@@ -0,0 +1,3 @@
+@org.osgi.annotation.bundle.Export
```

Comment:
We should not use `@Export` in this repo. Export from the bnd file.

Reply:
This was a cut-and-paste from `org.bndtools.api`. I have fixed all instances in the bundle, including `org.bndtools.api`.